CuratedContent/KanbanAndScrumMakingTheBestOfBoth.md

```diff
@@ -0,0 +1,61 @@
+# Kanban and Scrum: making the best of both
+
+<!--deck text start-->
+Scrum and Kanban are two of the best known lean/agile software project management frameworks.
+The mini-book *Kanban and Scrum: making the best of both* compares and contrasts these two process frameworks and breaks them down into a set of tools that can be mixed and matched to construct a wide range of hybrid processes.
+<!--deck text end-->
+
+#### Contributed by [Roscoe A. Bartlett](https://github.com/bartlettroscoe)
+
+#### Publication date: April ???, 2021
+
+
+Resource information | Details 
+:--- | :--- 
+Book title  | [Kanban and Scrum: making the best of both](https://www.infoq.com/mini-books/kanban-scrum-mini-book/)
```

Comment:
Link not working?

Reply:
I was able to confirm that the link did not work for me either from the GitHub rendered file.  Should be fixed in commit https://github.com/betterscientificsoftware/bssw.io/pull/783/commits/ed0b0abc0a6d7e2cca9a13d0eb9186d22f45c24f.  See updated file [KanbanAndScrumMakingTheBestOfBoth.md](https://github.com/bartlettroscoe/bssw.io/blob/576-scrum-vs-kanban/CuratedContent/KanbanAndScrumMakingTheBestOfBoth.md).